website/source/assets/stylesheets/_inner.scss

```diff
@@ -16,14 +36,12 @@
   code,
   pre code,
   tt {
-    font-family: $font-family-monospace;
-    font-size: $font-size - 2;
-    line-height: 1.6;
+    line-height: 1.926em;
```

Comment:
I think the line-height for the code blocks is a bit too big now. It's computed to 26px whereas previously it was 20.
Now:
![image](https://user-images.githubusercontent.com/1034429/67905633-3bfd4700-fb2f-11e9-82c3-f34f0948df09.png)


Before:
![image](https://user-images.githubusercontent.com/1034429/67905676-59caac00-fb2f-11e9-9f80-4b0663db5fd6.png)

Maybe 1.626?

![image](https://user-images.githubusercontent.com/1034429/67905775-b8902580-fb2f-11e9-94f4-53bee03ad6a7.png)